editor/inspector/editor_inspector.cpp

```diff
@@ -4619,6 +4619,21 @@ void EditorInspector::update_tree() {
 		}
 	}
 
+	//Lock Inspector Button	- By Aman Batra
```

Comment:
```suggestion
	// Lock Inspector Button.
```
We don't sign comments